pkg/util/billing/billing.go

```diff
@@ -165,9 +165,7 @@ func (m *manager) createOrUpdateE2EBlob(ctx context.Context, doc *api.BillingDoc
 	}
 
 	// The following is added to get rid of the '-' at the end in order to avoid an invalid container name.
-	if containerName[len(containerName)-1:] == "-" {
-		containerName = containerName[:len(containerName)-1]
-	}
+	strings.TrimSuffix(containerName, "-")
```

Comment:
@pkoushik @jim-minter Unfortunately, this will have no effect as `containerName` is not getting a new value assigned to it. It needs to be `containerName = strings.TrimSuffix(containerName, "-")`.

Reply:
I have just pushed up this change, I didn't realize it already got merged.